airflow/www/templates/airflow/code.html

```diff
@@ -22,22 +22,22 @@
 {% block page_title %}{{ dag.dag_id }} - Code - Airflow{% endblock %}
 
 {% block content %}
-    {{ super() }}
-    <h2>{{ title }}</h2>
```

Comment:
@ryanahamilton have you used some liting? Those changes look unrelated but I'm ok with them if we can add linter to pre-commit 👍

Reply:
There is no good linter for Jinja2 😿.

Reply:
Yeah, no linter. I'm just formatting as HTML files—indenting nested code blocks (w/ [2 spaces](https://github.com/apache/airflow/blob/master/.editorconfig#L48-L49)). There was very inconsistent formatting throughout the template files. Hopeful that introducing some consistency can at least suggest patterns to fellow contributors—even if they aren't enforceable w/ linting. It's kind of an obsessive behavior of mine driven by my Boy Scout days—leave your ~campsite~ code in better shape than you found it. 😃 

I will note that there is linting (esLint) in place for standalone JavaScript files that is currently not PR enforced (but did inform my formatting edits). We could/should enforce this, but the extent of the current offenses is so large that it should be addressed in a standalone PR.